thirdparty/build-thirdparty.sh

```diff
@@ -514,14 +514,23 @@ build_rocksdb() {
     export CFLAGS=$OLD_FLAGS
 }
 
+# sasl
+build_sasl() {
+    check_if_source_exist $SASL_SOURCE
+    cd $TP_SOURCE_DIR/$SASL_SOURCE
+    CFLAGS=  ./autogen.sh --prefix=$TP_INSTALL_DIR --enable-gssapi=no --enable-static=yes --enable-shared=no
```

Comment:
the format is not right. 
```suggestion
    CFLAGS=./autogen.sh --prefix=$TP_INSTALL_DIR --enable-gssapi=no --enable-static=yes --enable-shared=no
```